config/initializers/active_record_log_subscriber.rb

```diff
@@ -0,0 +1,16 @@
+module LogQuerySource
```

Comment:
This comes from http://www.jkfill.com/2015/02/14/log-which-line-caused-a-query/ 

And makes it significantly easier to find out where the issue is ( but not where is the right place to fix it )